### PR TITLE
Explain revisions and architectures in the context of overrides

### PR DESCRIPTION
**Pull request by suligap**
Fix for https://bugs.launchpad.net/snapstore-snap/+bug/1787624

### CODE REVIEW

File: en/overrides.md [hunk 1]
```diff
@@ -30,6 +30,46 @@ To remove all current overrides on a channel:
 
     sudo snap-proxy delete-override <snap> <channel>
 
+### Revisions and Architectures
+
+Every *.snap file has a unique
```
Comment (knitzsche) [inline en/overrides.md]: Perhaps say "A  Snap Store channel can publish only one revision of a specific snap at any time". This phrasing avoids the implication that every snap is published on every channel.

File: en/overrides.md [hunk 2]
```diff
@@ -30,6 +30,46 @@ To remove all current overrides on a channel:
...
+[revision](https://snapcraft.io/docs/getting-started) within a channel.
+
+One revision can support one or multiple architectures. Specifying a revision
```
Comment (knitzsche) [inline en/overrides.md]: perhaps say: "A snap revision can..."

File: en/overrides.md [hunk 3]
```diff
@@ -30,6 +30,46 @@ To remove all current overrides on a channel:
...
+for an override will therefore determine for which architectures the override is
```
Comment (knitzsche) [inline en/overrides.md]: A more usual phrasing/word order would be "Specifying a revision for an override therefore also determines the architectures involved."

File: en/overrides.md [hunk 4]
```diff
@@ -30,6 +30,46 @@ To remove all current overrides on a channel:
...
+created.
+
+Revisions for specific snaps can be looked up using the `snap info` command,
+which will list currently available revisions for the architecture of the device
```
Comment (knitzsche) [inline en/overrides.md]: don't use future tense ("will") Just say the command "lists currently available..."

File: en/overrides.md [hunk 5]
```diff
@@ -30,6 +30,46 @@ To remove all current overrides on a channel:
...
+$ sudo snap-proxy override core18 stable=1725
+core18 stable armhf 1725
+
+# We can see that we've overriden the stable channel revisions for both
+# amd64 and armhf and that both upstream counterparts ar at lower revisions.
+$ sudo snap-store-proxy list-overrides core18
+core18 stable amd64 1722 (upstream 1705)
+core18 stable armhf 1725 (upstream 1706)
+
+# Deleting a channel specific override will delete overrides for all revisions
```
Comment (knitzsche) [inline en/overrides.md]: * "channel-specific override" (you need the hyphen)
* don't use the future tense ("will")
So: "Deleting a channel-specific override deletes overrides for all revisions and architectures.

**Comment by suligap** [comment]
Thanks a lot @knitzsche! Pushed fixes based on your suggestions.

**Pull request merged**